src/useClickOutside.ts

```diff
@@ -0,0 +1,33 @@
+import { useCallback, useEffect } from 'react';
+
+export function useClickOutside(
```

Comment:
nit: I've generally seen this as a useOutsideClick hook 🤷

Reply:
updated